tasks/main.yml

```diff
@@ -1,63 +1,19 @@
 ---
+# manage dnf bases on the distribution
```

Comment:
I think the conditional import logic can be greatly simplified. As far as I know, `ansible_pkg_mgr` is either set to `dnf` for older dnf or to `dnf5`. This should be enough. Or do you see any problem with this approach?